pkgs/build-support/bintools-wrapper/add-hardening.sh

```diff
@@ -37,7 +37,7 @@ fi
 for flag in "${!hardeningEnableMap[@]}"; do
   case $flag in
     pie)
-      if [[ ! ("$*" =~ " -shared " || "$*" =~ " -static ") ]]; then
+      if [[ ! ("$*" =~ " -shared " || "$*" =~ " -static " || "$*" =~ " -r " || "$*" =~ " -Ur ") ]]; then
```

Comment:
maybe a case is better to read? On the other hand it is already very nested.

Reply:
yeah, i think it's okay for now, although perhaps one day we'll enable -pie by default. Right now only musl gets it.